### PR TITLE
BZ1839982 FOR 2.4: Updating access mode for LiveMigration

### PR DESCRIPTION
**Pull request by bgaydosrh**
Expanding content to include a separate module with prereqs and procedure for changing access mode from RWO to RWX to enable LiveMigration.

This BZ has been code and QE reviewed and approved and went through peer review extensively. 
Steve Bream is recommending it be merged, but feel free to review for suggestions, @adellape 

Requesting peer review from @adellape --- as this is a bug, please fit this in when you can as we can push async to the 2.4 build. 

Please label *peer review needed* and  *enterprise-4.5*

Test Build: See Netify build.

### CODE REVIEW

File: virt/live_migration/virt-live-migration.adoc [hunk 1]
```diff
@@ -4,7 +4,15 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-live-migration
 toc::[]
 
+.Prerequisites
+* Before using LiveMigration, ensure that the storage class used by the
+virtual machine has a persistent volume claim (PVC) with a shared
+ReadWriteMany (RWX) access mode.
+See xref:../../virt/virtual_machines/virtual_disks/virt-storage-defaults-for-datavolumes.adoc#virt-storage-defaults-for-datavolumes[Storage defaults for DataVolumes]
+to ensure all your storage settings are correct.
```
Comment (adellape) [inline virt/live_migration/virt-live-migration.adoc:12]: Suggest:

s/all//

"to ensure your storage settings are correct."

**Review by adellape — COMMENTED** [review]
Minor suggestions, but LGTM!

**Comment by vikram-redhat** [comment]
@bgaydosrh was this intended for the GA?

**Comment by bgaydosrh** [comment]
> @bgaydosrh was this intended for the GA?

Hi @vikram-redhat - Yes, and @adellape has peer reviewed and I only have to input his changes. Have had to prioritize bugs lower than changing feature work up until now. Plan is to have this complete and merged today. Have 1 other bug that is small. I was told to focus on release work and release notes first. 

cc: @sbream

File: modules/virt-updating-access-mode-for-live-migration.adoc [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[id="virt-updating-access-mode-for-live-migration_{context}"]
+= Updating access mode for LiveMigration
+
+For LiveMigration to function properly, you must use the
+ReadWriteMany (RWX) access mode. Use this
+procedure to update the access mode, if needed.
+
+.Procedure
+* To set the RWX access mode, run the following `patch` command.
```
[the remark this answers is not in the document]
Comment (bgaydosrh) [reply modules/virt-updating-access-mode-for-live-migration.adoc:13]: Done.

File: virt/live_migration/virt-live-migration.adoc [hunk 3]
```diff
@@ -4,7 +4,15 @@ include::modules/virt-document-attributes.adoc[]
 :context: virt-live-migration
 toc::[]
 
+.Prerequisites
+* Before using LiveMigration, ensure that the storage class used by the
+virtual machine has a persistent volume claim (PVC) with a shared
```
[the remark this answers is not in the document]
Comment (bgaydosrh) [reply virt/live_migration/virt-live-migration.adoc:9]: Done

File: virt/live_migration/virt-live-migration.adoc (hunk 1 above)
Comment (bgaydosrh) [reply virt/live_migration/virt-live-migration.adoc:12]: Done

File: modules/virt-updating-access-mode-for-live-migration.adoc [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
++
+----
+$ oc patch -n openshift-cnv cm kubevirt-storage-class-defaults -p '{"data":{"'$<STORAGE_CLASS>'.accessMode":"ReadWriteMany"}}'
```
[the remark this answers is not in the document]
Comment (bgaydosrh) [reply modules/virt-updating-access-mode-for-live-migration.adoc:16]: Done.

File: modules/virt-updating-access-mode-for-live-migration.adoc [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+[id="virt-updating-access-mode-for-live-migration_{context}"]
+= Updating access mode for LiveMigration
+
+For LiveMigration to function properly, you must use the
+ReadWriteMany (RWX) access mode. Use this
+procedure to update the access mode, if needed.
+
+.Procedure
+* To set the RWX access mode, run the following `oc patch` command.
```
Comment (adellape) [inline modules/virt-updating-access-mode-for-live-migration.adoc:13]: s/command./command:/

**Review by adellape — APPROVED** [review]
One tiny nit, but LGTM!

**Pull request merged**